packages/blade/src/tokens/global/sizes.ts

```diff
@@ -0,0 +1,79 @@
+export type Sizes = Readonly<{
+  /** 0: 0px */
+  0: 0;
+  /** 25: 1px */
+  25: 1;
+  /** 50: 2px */
+  50: 2;
+  /** 75: 3px */
+  75: 3;
```

Comment:
@divyanshu013 Can we know where are we using `3px` as a size token?

Reply:
In radio icon [dot radius](https://github.com/razorpay/blade/pull/1019/files#diff-6486bcbed9d9e376d3529b6ff2454df2dffc7d6e00f67dac8d2f03895a7dbd49R27)